pyproject.toml

```diff
@@ -35,8 +35,7 @@ pandas = "^2.1"
 rich = "^12.5.1"
 click = "^8.0.4"
 tomli = "^2.0.1"
-nnpdf = { git = "https://github.com/NNPDF/nnpdf", optional = true}
-lhapdf-management = { version = "^0.5", optional = true }
```

Comment:
Why remove the `lhapdf-management` dependency? Doesn't pineko itself also use it

Reply:
No, it was only nnpdf and only through imports.

Reply:
Ah so whenever pineko uses lhapdf, we need genuine lhapdf and can't use `lhapdf-management`?

Reply:
Yes.But I think it is only used if you choose to compare fktable and grid at the end (which maybe we should do by default but that's another topic)

Reply:
okay anyway replacing lhapdf with lhapdf-management is not the point of this PR